schemas/input_json_format.json

```diff
@@ -0,0 +1,143 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "http://example.com/example.json",
+  "type": "object",
+  "title": "The root schema",
+  "description": "https://github.com/vowpalwabbit/vowpal_wabbit/wiki/JSON",
+  "default": {},
+  "examples": [],
+  "required": [],
+  "definitions": {
+  },
+  "additionalProperties": true,
+  "patternProperties": {
+    "^[^_].*": {
+      "type": [
+        "string",
+        "number",
+        "boolean",
+        "array",
+        "object"
+      ],
+      "items": {
+        "type": "number"
+      }
+    },
+    "^_": {
+      "type": [
+        "number",
+        "string",
+        "boolean",
+        "object",
+        "array",
+        "null"
+      ]
+    }
+  },
+  "properties": {
+    "_tag": {
+      "$id": "#/properties/_tag",
+      "type": "string",
+      "title": "The _tag schema",
+      "description": "An explanation about the purpose of this instance.",
+      "default": "",
+      "examples": [
+        "mytag"
+      ]
+    },
+    "_label": {
+      "oneOf": [
+        {
+          "$id": "#/properties/_label",
+          "type": "integer",
+          "title": "The _label schema",
+          "description": "An explanation about the purpose of this instance.",
+          "default": 0,
+          "examples": [
+            1
+          ]
+        },
+        {
+          "$id": "#/properties/_labelobject",
+          "type": "object",
+          "title": "The _label schema",
+          "description": "An explanation about the purpose of this instance.",
+          "default": 0,
+          "oneOf": [
```

Comment:
This nested oneOf->anyOf is basically this line `if the JSON value is an object, the first property needs to match one of the JSON properties of SimpleLabel or ContextualBanditLabel.`. It's interesting to take a look at.

Reply:
Do we actually support this right now?